toplevel/opttoploop.ml

```diff
@@ -544,17 +544,42 @@ let _ =
   Clflags.dlcode := true;
   ()
 
+let find_ocamlinit () =
+  let ocamlinit = ".ocamlinit" in
+  if Sys.file_exists ocamlinit then Some ocamlinit else
+  let exists_in_dir dir p = match dir with
+  | None -> None
+  | Some dir ->
```

Comment:
Is the weird indentation intentional? I would expect the match cases to be indented (and below as well)

Reply:
That's the way `ocp-indent` idents for me (`strict_with=always`, a good way of making the most of the 80 columns that are available to express yourself) but I can certainly change that if it disturbs people.

Reply:
In fact I realized there's an `.ocp-indent` in the repo but apparently it wasn't taken into account. Let me fix that.

Reply:
Ok it seems there was `.emacs` configuration taking over on my side. Now indented according the the repo's `.ocp-indent`.

Reply:
@dbuenzli What Emacs configuration?  Tuareg indents `|` by default.  It also indents the `else` clause:
https://github.com/ocaml/ocaml/pull/8834/files#diff-59d1589c8c3e651da23f92e75ce9c12eR550-R572

Reply:
An `ocp-indent` emacs configuration. I don't user tuareg anyways but `caml-mode`.